website/blog/2018-07-02-releasing-react-native-056.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Releasing 0.56
+author: Lorenzo Sciandra
+authorTitle: React Native Developer and Core Maintainer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: engineering
+---
+
+If you are reading this, it means that the long awaited 0.56 version of React Native is, officially, released 🎉.
+
+This blog post highlights some of the [changes](https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#highlights) that this massive new version introduces and explains what we've been busy with since May.
+
+React Native changes go through a multi-step process to keep up with the internal Facebook team (read about it in the [How To Contribute Guide](https://facebook.github.io/react-native/docs/contributing#our-development-process)). It is also composed of [many different tools](https://github.com/facebook/react-native-website/issues/370) which require coordination and constant support to keep the framework working properly. Add the broad, vibrant community of developers in the open source world who contribute to make it better, and you will see the awesome scale of it all (It takes incredible effort to produce monthly stable versions!).
+
+With React Native's impressive adoption, breaking changes must be made with great care. Sometimes this process isn't as quick as desired. Couple that with inconvenient timing for many contributors, and a decision was forced to skip the April and May monthly releases. Instead, the team bundled the first 0.56 rc with the intent of simplifying the BC handling, and extra testing took place. [Dedicated community communication](https://github.com/react-native-community/react-native-releases/issues/14) channels were used along the way to ensure that 0.56.0 will be as hassle-free as possible to adopt by those who patiently waited for the stable release.
+
+Hopefully, these few paragraphs helped explain what we've been so busy with. Thanks to all the 62 contributors who worked on the massive 803 commits that will make your apps even better 👏
```

Comment:
Do you think it can be worthy to link the diff here? https://github.com/facebook/react-native/compare/v0.55.4...v0.56.0-rc.4